service/docs/source/devel.rst

```diff
@@ -1,197 +1,123 @@
+GEOPM Developer Guide
```

Comment:
No discussion of the runtime RPMs and process...

Reply:
Open an issue to track refactoring the runtime.rst and devel.rst around OBS RPMs, Launchpad DEBs, and MPI support "requirements".